mktags.bash

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Icinga 2 Docker image | (c) 2023 Icinga GmbH | GPLv2+
+set -exo pipefail
+
+if [[ "$1" =~ ^v((([0-9]+).([0-9]+)).[0-9]+)$ ]]; then
+  BUILDX=(docker buildx build --platform "$(cat "$(realpath "$(dirname "$0")")/platforms.txt")" --push)
+  cd "$(mktemp -d)"
+
+  cat <<<"FROM icinga/icinga2:${BASH_REMATCH[1]}" >Dockerfile
+
+  "${BUILDX[@]}" -t "icinga/icinga2:${BASH_REMATCH[2]}" .
+
+  docker pull "icinga/icinga2:${BASH_REMATCH[2]}"
+
+  if ! docker pull "icinga/icinga2:${BASH_REMATCH[3]}.$((${BASH_REMATCH[4]}+1))"; then
+    "${BUILDX[@]}" -t "icinga/icinga2:${BASH_REMATCH[3]}" .
+    "${BUILDX[@]}" -t icinga/icinga2 .
+  fi
+fi
```

Comment:
Splitting the version into three variables would probably be more readable than nested match groups and using `BASH_REMATCH` directly.